Dockerfile

```diff
@@ -55,6 +55,15 @@ WORKDIR /
 
 ENTRYPOINT ["/gardenlet"]
 
+############# admission-controller #############
```

Comment:
Just a reminder that we will need to adapt the `meta/ci` ref

Reply:
Thanks, will do as soon as this PR is merged.